### ISSUE TITLE
Request: separate physical table name for a model

### ISSUE THREAD

**Issue by henri-if** [datacontract/datacontract-cli#270]
Hi,

Would it be possible to implement a way to specify physical table name for a model separately from the "logical" model name?

When making breaking changes to data products, we often want to provide multiple major versions of a data product in parallel, in order to allow downstream consumers to do a controlled migration at their own schedule. Inspired by the dbt approach to versioning models, we'd prefer to do this by providing tables with major version as the suffix, for example sales_v1 and sales_v2.

Currently specifying data contracts for such tables would mean that the model names would have to include the major version, breaking the diff functionality.

I discussed this with Simon, and the suggested solution is to add a new config field for models:
```
models:
  my-model:
    config:
      databricksTable: my-table-v2
```
The physical table name could then be used when running tests, and should likely be used in at least some of the exports.

**Comment by jochenchrist** [comment]
Agree, that is a good idea, and it fits well with the "databricksType" config option on field level.

**Comment by jochenchrist** [comment]
This is now available with ODCS.
(Not sure if the schema.physicalName is respected by the CLI, but this would then be considered as a bug)
